pkg/ibmvpc/examples/out/analysis_out/filters_split_lb_subnet_all_vpcs__no_lbAbstract.txt

```diff
@@ -36,70 +15,91 @@ alb[LB private IP][10.240.65.4] => vsi0-sub3[10.240.128.4] : All Connections
 alb[LB private IP][10.240.65.4] => vsi1-sub1[10.240.0.4] : All Connections
 alb[LB private IP][10.240.65.4] => vsi1-sub2[10.240.64.4] : All Connections
 alb[LB private IP][10.240.65.4] => vsi1-sub3[10.240.128.5] : All Connections
+alb[Potential LB private IP][10.240.1.0/25] => vsi0-sub1[10.240.0.5] : All Connections
```

Comment:
The src pip are not the same as before.
E.g.: 10.240.1.4 before vs 10.240.1.0/25
Is this intended?
And why is it?
How is it related to adding LB to the cli?

Reply:
yes, it is intended. its not relevant to the cli.
it make more sense for fake/potenetial private IPs to print the block that they are created for, and not the specific address that was chosen for them